test/framework/installer.go

```diff
@@ -0,0 +1,32 @@
+package framework
+
+import "strings"
+
+// InstallerType defines the type of installer for services
+type InstallerType struct {
+	Name string
```

Comment:
Maybe changing to:

```go
type InstallerType string

const (
    CLIInstallerType InstallerType = "cli"
    CRInstallerType InstallerType = "cr"
)
```

Since you only have a `Name` attribute, it's not worth to have a structure in this case. :)

Reply:
I needed to be able to display the installer name if it was unknown. But I will panic instead (like ParseBool)